### ISSUE TITLE
Consider stripping core IATI standard repositories back to core XSD, XML and text

### ISSUE THREAD

**Issue by andylolz** [IATI/IATI-Standard-SSOT#468]
**Is your feature request related to a problem? Please describe.**
I’d like to follow repositories that contain the core IATI standard (and non-embedded codelists), so that I can easily see proposed and implemented patch changes; codelist changes etc.

Currently that’s difficult, because these repos contain a lot of downstream content that are regularly modified. E.g. code used by datastore search; reference site generator; validator etc.

**Describe the solution you'd like**
Ideally these core IATI standard repositories would be as lean as possible, so that any changes made to the core standard can easily be tracked, e.g. by organisations reliant on the standard. Anything superfluous should be considered carefully before adding.

It’s good practice to separate concerns in this way, so that upstream repositories don’t contain code that’s utilised by specific downstream products.

**Additional context**
Code for generating the IATI reference site has lived in these repos for A Long Time. I have previously suggested that it shouldn’t (see e.g. #227) and have discussed this at length with past IATI tech teams. It seems as though more and more downstream code is being added to these core repositories, making it harder and harder to keep track of changes to the core standard. This feels to me like the wrong direction of travel.

**Comment by akmiller01** [comment]
Agreed this would be a good change. Especially considering we now have the entire HTML generation process running off of a single Python environment in this repository, this could be accomplished by moving the make-files up out of the submodules: https://github.com/IATI/IATI-Reference-Generator

**Comment by andylolz** [comment]
> Especially considering we now have the entire HTML generation process running off of a single Python environment [in this repository](https://github.com/IATI/IATI-Reference-Generator), this could be accomplished by moving the make-files up out of the submodules

+1, I think this would be an excellent change.
